fiwaremurano/meta/io.murano/Resources/support.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+same_uuid() {
+  curl 'http://169.254.169.254/openstack/latest/meta_data.json' --output /etc/metadata.json
+  if [ -f "/etc/metadata.json" ]
+  then
+    if grep -Po '"uuid":.*?[^\\]",' /etc/metadata.json | grep `cat /etc/fiware-support/uuid`; then
```

Comment:
If you have a response but this response is not matched with the expected value to extract the uuid, what do you do?